pages/configuration/configuration-settings.mdx

```diff
@@ -9,32 +9,182 @@ import { Steps } from 'nextra/components'
 
-If you are working with Memgraph Platform Docker image, you should pass
-configuration flags inside of environment variables, e.g. `docker run -p
-7687:7687 -p 7444:7444 -p 3000:3000 -e MEMGRAPH="--bolt-port=7687
---log-level=TRACE" memgraph/memgraph-platform`.
+You can check the current configuration by using the following query:
 
-</Callout>
+```opencypher
+SHOW CONFIG;
+```
 
-Each configuration setting is in the form: `--setting-name=value`.
+## Changing configuration settings
+
+The `memgraph.conf` file is the persistency for configuration, and the way you
+can change the configuration settings depending on the way you are using Memgraph
+and configuration settings you want to change.
+
+Most of the configuration changes need to happen before Memgraph is started, 
+there is a set of configuration settings that can be changed during **runtime**. 
+
+### Change configuration settings on startup
+
+Changing the configuration settings for Memgraph differs if you are using
+Memgraph with **Docker**, **Docker Compose**, or if it was installed on the native **Linux**.
+
+<Tabs items={["Docker", "Docker Compose" , "Linux"]}>
+
+    <Tab>
+
+        You can customize the configuration by
+        passing configuration options within the `docker run` command.
+
+        When you are working with `memgraph` or `memgraph-mage` images, you should pass
+        configuration options as arguments.
+
+        For example, if you want to limit memory usage for the whole instance to 50 MiB
+        and set the log level to `TRACE`, pass the configuration argument like this:
+
+        ```
+        docker run -p 7687:7687 -p 7444:7444 memgraph/memgraph --memory-limit=50 --log-level=TRACE
+        ```
+        ---
+
+        You can also change the configuration within the configuration file:
+
+        <Steps>
+
+            {<h3>Start Memgraph</h3>}
+
+            Start Memgraph with a `docker run` command.
+
+            {<h3>Find container ID</h3>}
+
+            Open a new terminal and find the `CONTAINER ID` of the Memgraph Docker container
+            using the following command:
+
+            ```console
+            docker ps
+            ```
+
+            {<h3>Enter the continer</h3>}
+
+            Enter the Docker container with the following command:
+
+            ```plaintext
+            docker exec -it -u 0 <CONTAINER ID> bash
+            ```
+
+            {<h3>Install the text editor of your choice</h3>}
+
+            For example, if you want to use `vim` run:
+
+            ```console
+            apt-get update
+            apt-get install vim
+            ```
+
+            {<h3>Edit the configuration file</h3>}
+
+            The file is located at `/etc/memgraph/memgraph.conf`.
+
+            {<h3>Restart the intance</h3>}
+
+            Run the following command:
+
+            ```
+            docker restart <CONTAINER ID>
+            ```
+
+        </Steps>
+
+    </Tab>
+
+    <Tab>
+    TBD docker-compose
```

Comment:
Remove this tab for now. We will add it later and we will be able to merge your PR.

Reply:
We agreed on daily to let this fly, so marking as resolved.